register/murmur3/multihash_murmur3.go

```diff
@@ -0,0 +1,40 @@
+/*
+	This package has no purpose except to perform registration of multihashes.
+
+	It is meant to be used as a side-effecting import, e.g.
+
+		import (
+			_ "github.com/multiformats/go-multihash/register/murmur3"
+		)
+
+	This package registers multihashes for murmur3
+*/
+package murmur3
+
+import (
+	"hash"
+
+	multihash "github.com/multiformats/go-multihash/core"
+	"github.com/spaolacci/murmur3"
+)
+
+func init() {
+	multihash.Register(multihash.MURMUR3X64_64, func() hash.Hash { return murmur64{murmur3.New64()} })
+}
+
+// A wrapper is needed to export the correct size, because murmur3 incorrectly advertises Hash64 as a 128bit hash.
+type murmur64 struct {
+	hash.Hash64
+}
+
+func (murmur64) BlockSize() int {
+	return 1
```

Comment:
this looks slightly funny.  I can't really fathom any harm to it, though.

... but I went on a splunk anyway, and I'd say 16 is probably an accurate number.  [citation](https://github.com/spaolacci/murmur3/blob/539464a789e9b9f01bc857458ffe2c5c1a2ed382/murmur128.go#L66)